import-export-cli/cmd/exportAPI.go

```diff
@@ -87,6 +87,7 @@ func executeExportAPICmd(credential credentials.Credential, exportDirectory stri
 		utils.Logf(utils.LogPrefixInfo+"ResponseStatus: %v\n", resp.Status())
 		apiZipLocationPath := filepath.Join(exportDirectory, CmdExportEnvironment)
 		if resp.StatusCode() == http.StatusOK {
+			fmt.Println("API resp: ", resp)
```

Comment:
Do we need this?

Reply:
Fixed by [02a788a](https://github.com/wso2/product-apim-tooling/pull/924/commits/02a788a1570035b84d7acfea4bc999161a1bae47)